api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -51,7 +51,8 @@
             + "zookeeper.connect, zookeeper.set.acl, authorizer., super.user"
             + "cruise.control.metrics.topic, cruise.control.metrics.reporter.bootstrap.servers";
 
-    public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "zookeeper.connection.timeout.ms, ssl.cipher.suites, ssl.protocol, ssl.enabled.protocols";
+    public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "zookeeper.connection.timeout.ms, ssl.cipher.suites, ssl.protocol, ssl.enabled.protocols, "
+            + "cruise.control.metrics.topic.num.partitions, cruise.control.metrics.topic.replication.factor";
```

Comment:
What about other settings? Retention? Segment size? etc.? If we assume the user configures these topics, there are many options which migght make sense. So shouldn't the whole `cruise.control.metrics.topic.` segment be user-configurable?

Reply:
Updated so the user is free to configure these things

Reply:
I had to double back and forbid `cruise.control.metrics.topic` as it is used to change the name of the CC metric topic. I have added all of the other `cruise.control.metrics.topic` options to the exception list, there are only three of them: `cruise.control.metrics.topic.num.partitions`, `cruise.control.metrics.topic.replication.factor`, `cruise.control.metrics.topic.retention.ms`